storage_service/storage_service/settings/base.py

```diff
@@ -386,7 +386,15 @@ def _get_settings_from_file(path):
         )
 
     # https://pythonhosted.org/django-auth-ldap/groups.html#types-of-groups
-    AUTH_LDAP_GROUP_TYPE = ldap_config.ActiveDirectoryGroupType()
+    if "AUTH_LDAP_GROUP_TYPE" in environ:
```

Comment:
This is not part of the issue, but when testing the GROUP settings it is needed to use a different ldap schema than the Active Directory. Probably we need the same in the Dashboard LDAP settings.